docs/elasticsearch-spec.asciidoc

```diff
@@ -100,3 +100,29 @@ Example to create a Kubernetes TLS secret with a self-signed certificate:
 $ openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -nodes
 $ kubectl create secret tls my-cert --cert tls.crt --key tls.key
 ----
+
+[id="{p}-virtual-memory"]
+=== Virtual memory
+
+By default, Elasticsearch is using memory mapping (mmap) to efficiently access indexes. Usually, default values for virtual address space on Linux distributions are too low for Elasticsearch to work properly, which may result in out of memory exceptions. To increase virtual memory ECK sets the recommended value by default. This requires that the kubelet is allowed to run privileged containers.
```

Comment:
I think ES doc uses the `indices` plural form instead of `indexes`.

Reply:
Fixed